packages/analitics/types.ts

```diff
@@ -0,0 +1,40 @@
+import { Package, Action } from "frontity/types";
+import Source from "@frontity/source";
+
+export type Pageview = {
+  page: string;
+  title: string;
+};
+
+export type Event = {
+  category: string;
+  action: string;
+  label: string;
+};
+
+interface Analytics extends Package {
+  actions: {
+    analytics: {
+      afterCSR: Action<Analytics>;
+      sendPageview: Action<Analytics, Pageview>;
+      sendEvent: Action<Analytics, Event>;
+    };
+    [key: string]: {
+      sendPageview: Action<Analytics, Pageview>;
+      sendEvent: Action<Analytics, Event>;
+    };
+  };
+  state: {
+    analytics: {
+      namespaces: Record<string, string>;
+    };
+    router?: {
+      link: string;
```

Comment:
I'm yet not sure if it's a better idea to hardcode external types like this (for the sake of simplicity) or to recommend importing the original type like this.

```js
router?: {
  link: Router["state"]["router"]["link"];
}
```

One scenario when this could be relevant is if you are updating this package to be used with `@frontity/router@^2.0.0` and in that version `router` has changed `link` from a string to an object for example. It's not a big deal.

Any other scenarios where this could be more important?